packages/ethereum-contracts/contracts/utils/SuperfluidFrameworkDeployer.sol

```diff
@@ -2,10 +2,6 @@
 pragma solidity ^0.8.0;
```

Comment:
Separate PR for this: https://github.com/superfluid-finance/protocol-monorepo/issues/1220